[Filter field] not displaying suggestions in MS Edge

# Bug Report

<!---
Hi there!

You are in the process of reporting a bug in Barista. To ensure a quick
turnaround, please enter/change/update the following fields:
-->
When clicking into the filter field in the non-chromium version of MS Edge, no filter suggestions are shown. Also, when typing, the clear all button does not appear. Instead there is a rather bad-looking "X" that clears the input (see screenshot).

This only happens, when the app is compiled with View Engine instead of Ivy though.

## Expected Behavior

<!--- Tell us what should happen -->
When clicking into the filter field, suggestions should be shown in a drop-down component, as in other browsers.

## Current Behavior

<!--- Tell us what happens instead of the expected behavior -->
Clicking into the filter field in MS Edge does not open a drop-down component with the filter suggestions. In place of the drop-down, a small border can be seen (see screenshot).

## Steps to Reproduce

<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->

1. Disable Angular Ivy in the app's tsconfig:
```
  "angularCompilerOptions": {
    "enableIvy": false
  }
```
2. Click into the filter field in any non-chromium version MS Edge.
3. No suggestion values are displayed

## Context (Environment)

<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

### Used Versions:

- **angular**: 10.0.9
- **@angular/cdk**: 10.1.1
- **@dynatrace/barista-components**: 8.0.0

<!-- Insert further versions if they are important -->

## Attachments

### Strange X instead of "clear all"
![image (5)](https://user-images.githubusercontent.com/35682879/92231814-b3b5a400-eead-11ea-883e-f46a9a74a241.png)

### No dropdown but border visible
![image (4)](https://user-images.githubusercontent.com/35682879/92231816-b3b5a400-eead-11ea-8268-05d74d260de3.png)

### Expected behaviour Firefox
![image (7)](https://user-images.githubusercontent.com/35682879/92231818-b44e3a80-eead-11ea-9a67-04b7da8a00a1.png)

### Actual behaviour MS Edge
![image (6)](https://user-images.githubusercontent.com/35682879/92231819-b4e6d100-eead-11ea-9ab8-821f0e5ffc5b.png)


<!-- Not obligatory, Append a screenshot, and any other visuals and required files if possible. -->

<!--
We at Dynatrace have a bugs first policy. Meaning, we will handle the bug as
soon as possible, depending on the severity of the bug.

Thank you and all the best! — The Dynatrace DesignOps team
-->

@lukasholzer Are you using the samples on https://barista.dynatrace.com/? because they work fine for me in Edge 18 too.
However, the problem only occurs with the new version 8.0.0 of barista components, and the examples on https://barista.dynatrace.com/ do not use v8.0.0.

I am currently not able to build a local example with the filter field and barista 8.0.0, as i always get an error message when building a fresh project with only a filter field on the page:
```
ERROR in node_modules/@dynatrace/barista-components/button-group/src/button-group.d.ts:35:9 - error TS2611: 'disabled' is defined as a property in class 'HasTabIndex & DtButtonGroupBase', but is overridden here in 'DtButtonGroup<T>' as an accessor.
35     get disabled(): boolean;
           ~~~~~~~~
node_modules/@dynatrace/barista-components/button-group/src/button-group.d.ts:76:9 - error TS2611: 'disabled' is defined as a property in class 'HasTabIndex & CanColor<DtButtonGroupThemePalette> & DtButtonGroupItemBase', but is overridden here in 'DtButtonGroupItem<T>' as an accessor.
76     get disabled(): boolean;
           ~~~~~~~~
```
Should i file another bug for this issue?

Also, i can not provide a stackblitz example because it does not work at all with MS Edge.

--- a/libs/barista-components/filter-field/src/filter-field.spec.ts
+++ b/libs/barista-components/filter-field/src/filter-field.spec.ts
@@ -499,8 +499,10 @@ describe('DtFilterField', () => {
 
       options = getOptions(overlayContainerElement);
       expect(options.length).toBe(2);
-      expect(options[0].textContent!.trim()).toBe('Los Angeles');
-      expect(options[1].textContent!.trim()).toBe('San Fran');
+      // We use contain in favor of toBe as the text has to be inside twice due to the highlight
+      // The second occurrence is hidden by display:none
+      expect(options[0].textContent).toContain('Los Angeles');
+      expect(options[1].textContent).toContain('San Fran');
 
       zone.simulateZoneExit();
     }));
@@ -524,7 +526,9 @@ describe('DtFilterField', () => {
 
       options = getOptions(overlayContainerElement);
       expect(options.length).toBe(1);
-      expect(options[0].textContent!.trim()).toBe('Upper Austria');
+      // We use contain in favor of toBe as the text has to be inside twice due to the highlight
+      // The second occurrence is hidden by display:none
+      expect(options[0].textContent).toContain('Upper Austria');
       tick();
     }));
 
@@ -786,7 +790,9 @@ describe('DtFilterField', () => {
       advanceFilterfieldCycle();
 
       options = getOptions(overlayContainerElement);
-      expect(options[0].textContent).toMatch(/^\s*AUT\s*$/);
+      // We use contain in favor of toBe as the text has to be inside twice due to the highlight
+      // The second occurrence is hidden by display:none
+      expect(options[0].textContent).toContain('AUT');
     });
 
     it('should switch back from root after deleting a unfinished freetext filter with BACKSPACE', () => {
@@ -803,9 +809,11 @@ describe('DtFilterField', () => {
       expect(tags.length).toBe(0);
 
       const options = getOptions(overlayContainerElement);
-      expect(options[0].textContent).toMatch(/^\s*AUT\s*$/);
-      expect(options[1].textContent).toMatch(/^\s*USA\s*$/);
-      expect(options[2].textContent).toMatch(/^\s*Free\s*$/);
+      // We use contain in favor of toBe as the text has to be inside twice due to the highlight
+      // The second occurrence is hidden by display:none
+      expect(options[0].textContent).toContain('AUT');
+      expect(options[1].textContent).toContain('USA');
+      expect(options[2].textContent).toContain('Free');
     });
 
     it('should remove a parent from an autocomplete if it is distinct and an option has been selected', () => {

--- a/libs/barista-components/highlight/src/highlight.ts
+++ b/libs/barista-components/highlight/src/highlight.ts
@@ -30,7 +30,6 @@ import {
   Optional,
   ViewChild,
   ViewEncapsulation,
-  AfterContentInit,
 } from '@angular/core';
 import { Subscription } from 'rxjs';
 import { take } from 'rxjs/operators';
@@ -78,12 +77,7 @@ function escapeRegExp(text: string): string {
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class DtHighlight
-  implements
-    AfterContentChecked,
-    AfterContentInit,
-    AfterViewInit,
-    OnChanges,
-    OnDestroy {
+  implements AfterContentChecked, AfterViewInit, OnChanges, OnDestroy {
   /**
    * The caseSensitive input can be set to search for case sensitive occurrences.
    * Per default the search is case insensitive.
@@ -140,6 +134,14 @@ export class DtHighlight
   }
 
   ngAfterViewInit(): void {
+    // Initially we need to run and force highlight once
+    // to move the text content value into the visible span
+    // Otherwise some layouts will be tripped up, as the visible span
+    // would be 0x0 pixels large.
+    const textContent = this._getTextContent();
+    this._textContent = textContent;
+    this._highlight(true);
+
     // Observable whether the component is in the viewport.
     this._isInViewportSubscription = createInViewportStream(
       this._elementRef,
@@ -150,16 +152,6 @@ export class DtHighlight
         this._highlight();
       }
     });
-  }
-
-  ngAfterContentInit(): void {
-    // Initially we need to run and force highlight once
-    // to move the text content value into the visible span
-    // Otherwise some layouts will be tripped up, as the visible span
-    // would be 0x0 pixels large.
-    const textContent = this._getTextContent();
-    this._textContent = textContent;
-    this._highlight(true);
   }
 
   ngOnDestroy(): void {

